### PR TITLE
Improve commit contents in trac issue

### PR DESCRIPTION
**Pull request by haocheng**
I would like to add more contents when trac-publisher plugin add comment in the related issue. Basically I copied the comment format from jia plugin (https://wiki.jenkins-ci.org/display/JENKINS/JIRA+Plugin)

### CODE REVIEW

**Comment by haocheng** [comment]
Hi, 

Any updates on this pull request?
Please let me know if any issues.
Thank you!

**Comment by batkinson** [comment]
Thanks for submitting this. I would like to try and integrate the changes, but to help me do that can you:
- Submit pull requests with a branch for each change (one for logging changes, one for comments, one for others)
- Minimize diff noise by not formatting lines you didn't change.

I used the default Eclipse built-in formatting for the project files. Until that changes, can you either use that format or only format lines you changed? It makes it extremely difficult to review what you're changing otherwise.

**Comment by batkinson** [comment]
Sorry, I didn't mean comments... I meant logging changes.

**Comment by batkinson** [comment]
Forget what I just said, I said it right the first time. I forgot this ticket was for the comment patterns.

File: src/main/java/jenkins/plugins/tracpublisher/TracPublisher.java [hunk 1]
```diff
@@ -73,64 +85,201 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher,
...
-			BuildListener listener, Integer issue) throws MalformedURLException {
+			Integer issue) throws MalformedURLException {
 		try {
 			String buildDN = build.getFullDisplayName();
 			String buildUrl = build.getUrl();
-			listener.getLogger().format(
-					"Updating corrected issue %d with %s\n:", issue, buildDN);
-			updateIssue("Referenced in unsuccessful builds prior to", issue,
-					buildDN, buildUrl);
+			LOGGER.info(String.format(
```
Comment (batkinson) [inline src/main/java/jenkins/plugins/tracpublisher/TracPublisher.java:124]: It looks like a copy/paste error here. A "corrected" issue is one that is referenced from SCM comments in a failed build, and are posted on the next successful build.

**Comment by batkinson** [comment]
Thank you for submitting the request. I'm not inclined to pull it based on where this is at, but I am working on implementing the essential functionality you're looking for.

**Comment by haocheng** [comment]
Cool, thank you :)

**Pull request closed without merging**